.env.local

```
@@ -7,4 +7,6 @@ ASTRO_WEB_PORT=4321
 DATABASE_URL=postgresql://mse_radar:mse_radar@localhost:5432/mse_radar
 
 BETTER_AUTH_SECRET=w28jHK7syJoESSjjL2cQ18iYl4M97ck8
-BETTER_AUTH_URL=http://localhost:4321
\ No newline at end of file
+BETTER_AUTH_URL=http://localhost:4321
+
+RESPONSE_ENCRYPTION_KEY=YjKXKOG9+vboYStao6kTbk0qXRKDSDIhYV6xfciSbSo=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)